packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -88,10 +106,25 @@ const ActionHeader = ({
   onPrevious,
   onNext,
   children,
+  backButtonA11yLabel,
+  nextButtonA11yLabel,
+  prevButtonA11yLabel,
   ...customProps
 }) => {
   const theme = React.useContext(ThemeContext);
 
+  const closeButtonId = `terra-action-header-close-button-${uniqueid()}`;
+  const maximizeButtonId = `terra-action-header-maximize-button-${uniqueid()}`;
+  const minimizeButtonId = `terra-action-header-minimize-button-${uniqueid()}`;
```

Comment:
Lets use same Id for all.
```suggestion
  const buttonId = uniqueid();
  const closeButtonId = `terra-action-header-close-button-${buttonId}`;
  const maximizeButtonId = `terra-action-header-maximize-button-${buttonId}`;
  const minimizeButtonId = `terra-action-header-minimize-button-${buttonId}`;
```